test/cmdlineargs.jl

```diff
@@ -597,6 +597,34 @@ let exename = `$(Base.julia_cmd()) --startup-file=no --color=no`
             "Int(Base.JLOptions().fast_math)"`)) == JL_OPTIONS_FAST_MATH_DEFAULT
     end
 
+    # --pkgimages with multiple cpu targets
+    @testset let v = readchomperrors(`$exename
```

Comment:
This probably needs a guard for x86_64 or a different list for supported architectures.